src/view/event-view.js

```diff
@@ -8,20 +9,29 @@ const createEventTemplate = (point) => {
     type,
     dateFrom,
     dateTo,
+    offers,
     isFavorite,
-    offers
   } = point;
 
+  const choisenOptions = eventOffers(type).offers;
```

Comment:
Можно лучше: с использованием деструктуризации
```suggestion
  const { offers } = getEventOffers(type);
```